metrics.json

```diff
@@ -2,6 +2,7 @@
   "SELECT host, ROUND(SUM(memory_size_in_total)/1024/1024) column_tables_used_mb FROM sys.m_cs_tables GROUP BY host;":
   {
     "enabled": true,
+    "hana_version": "1.0",
```

Comment:
The patch level is relevant here. 2.00.010 version has much fewer views and features compared with 2.00.030, so we need to be able to organize the queries on this granularity.

Reply:
In this case everything above 1.0 would be executed (1.0.x included)